yang/frr-bfdd.yang

```diff
@@ -0,0 +1,387 @@
+module frr-bfdd {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/bfdd";
+  prefix frr-bfdd;
```

Comment:
before going further, I would like to be sure that the way BFD sessions are managed is ok with vrf.
that is to say, that contrary to some other routing services, where vrfname is part of the pathname, here, bfd does not mention the vrfname keyword in its path.

I would like to be sure that this is what we want.

later on, that implies that some other monitoring services would not be indexed per vrf.